test/python/quantum_info/states/test_statevector.py

```diff
@@ -1152,6 +1152,37 @@ def test_expval_pauli_qargs(self, qubits):
         expval = state.expectation_value(op, qubits)
         self.assertAlmostEqual(expval, target)
 
+
+    def test_expval_identity(self):
```

Comment:
It seems like you need to run the formatter here, you can do that e.g. by calling `make black` (you might want to make sure the right version is installed before doing so, by running `pip install -r requirements-dev.txt -c constraints.txt` before)

Reply:
Got it. Thanks for the instruction.